modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/internal/upgrade/v1_0_1/RemoteAppEntryUrlUpgradeProcess.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.remote.app.internal.upgrade.v1_0_1;
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.remote.app.internal.upgrade.v1_0_1.util.RemoteAppEntryTable;
+
+/**
+ * @author Iván Zaera Avellón
+ */
+public class RemoteAppEntryUrlUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		alter(
+			RemoteAppEntryTable.class,
+			new AlterColumnType("url", "VARCHAR(1024) null"));
```

Comment:
Did you apply this even for remote apps, nice :)

Reply:
Yes, 75 was way too small.